yarn.lock

```diff
@@ -7113,10 +7113,10 @@ svelte-loader@^3.1.2:
     svelte-dev-helper "^1.1.9"
     svelte-hmr "^0.14.2"
 
-svelte-materialify@^0.3.6:
-  version "0.3.6"
-  resolved "https://registry.yarnpkg.com/svelte-materialify/-/svelte-materialify-0.3.6.tgz#a2d4c42ddfcca4bd36625b0481c8daa62dae3b4d"
-  integrity sha512-tiENt0PkgIGa2alIAu3pAVUakQwNpWpxiudUDNyVo0igDuvPzXy/39R+wvPdM4HabjYUdsfoghiOsUcpHCLxIg==
+svelte-materialify@^0.3.11:
+  version "0.3.11"
+  resolved "https://registry.yarnpkg.com/svelte-materialify/-/svelte-materialify-0.3.11.tgz#bdd0a73961e9b408bb584bf2d85b2bb39e5424e9"
+  integrity sha512-MLtZtdSOHditOdJOYqlB8s7NxIFRbYKXJzij5s9gz5WVbJFVEqdX2TqOleg4wYeMOujtOCm+/x3RP5YGyzRxIQ==
```

Comment:
Please revert this back to materialify-0.3.6, stuff is broken with our project and materialify-0.3.11